ubuntu_image/builder.py

```diff
@@ -70,6 +73,27 @@ def _calculate_dirsize(self, path):
         total *= 1.5
         return total
 
+    def _mke2fs_has_dash_d(self):
```

Comment:
I wonder if we should trust version numbers?  Maybe we should just call it with `-d` and fallback (and cache) if that fails?